docs/docs.go

```diff
@@ -9,7 +9,13 @@ import (
 )
 
 func main() {
-	err := docs.GenerateDocs(resources.Provider(), "./docs")
+	outputPath := "./docs"
+	err := os.RemoveAll(outputPath)
+	if err != nil {
+		fmt.Printf("%+v", err)
```

Comment:
Shouldn't it exit with non nil if err ?

Reply:
I think the next err is also missing `os.exit(1)`

Reply:
Agreed